sample_factory/benchmarking/benchmarking.txt

```diff
@@ -467,6 +467,14 @@ python -m sample_factory.algorithms.appo.train_appo --env=doom_benchmark --algo=
 [2022-01-09 05:20:27,264][184143] Policy worker avg. requests 6.32, timing: init: 2.7602, wait_policy_total: 15.4553, wait_policy: 0.0052, handle_policy_step: 29.2352, one_step: 0.0025, deserialize: 3.5863, stack: 0.2062, obs_to_device: 7.6138, forward: 9.5064, to_cpu: 3.3929, format_outputs: 0.9884, postprocess: 2.5917, weight_update: 0.0006
 [2022-01-09 05:20:27,366][184116] GPU learner timing: extract: 0.1750, buffers: 0.0567, batching: 5.2542, buff_ready: 0.1724, tensors_gpu_float: 1.6330, squeeze: 0.0064, prepare: 7.1875, batcher_mem: 5.1569
 [2022-01-09 05:20:27,673][184116] Train loop timing: init: 1.9581, train_wait: 0.3825, epoch_init: 0.0011, minibatch_init: 0.0006, forward_head: 0.2330, bptt_initial: 2.9500, bptt_forward_core: 0.2780, bptt: 0.2895, tail: 0.1421, vtrace: 0.6693, losses: 0.4309, clip: 0.2765, update: 1.8503, after_optimizer: 0.3446, train: 7.3772
-[2022-01-09 05:20:27,801][184052] Workers joined!
 [2022-01-09 05:20:27,810][184052] Collected {0: 2015232}, FPS: 48606.9
-[2022-01-09 05:20:27,811][184052] Timing: experience: 41.2913
\ No newline at end of file
+[2022-01-09 05:20:27,811][184052] Timing: experience: 41.2913
+
+V1.123.0 with max entropy and GAE
```

Comment:
outside of this benchmarking, will you add release notes with this PR?

Reply:
yes!